docs/deployment/README.md

```diff
@@ -249,6 +249,8 @@ In `terraform.tf`, configure the remote state settings by substituting the appro
 
 Fill in the appropriate values in `terraform.tfvars`. See the [data-persistence module variable definitions](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence/variables.tf) for more detail on each variable.
 
+Consider [the size of your Elasticsearch cluster](#elasticsearch) when configuring data-persistence.
```

Comment:
Is there a reason to not include the sizing information directly in the relevant sections? I worry about it getting buried at the bottom of the document, though it is linked. I don't feel that strongly either way

Reply:
So historically the documentation has gotten out of control and it makes it hard for users who want to just get straight to deployment, especially in sandbox.